setup.cfg

```
@@ -31,7 +31,7 @@ concurrency = thread,multiprocessing
 parallel = True
 source = signac
 omit = 
-	*/signac/common/configobj/*.py
+	*/signac/_vendor/configobj/*.py
```

Can we make this broader?
```suggestion
	*/signac/_vendor/*
```